pages/rules.html

```diff
@@ -43,31 +44,29 @@
 				{% endfor %}
 			</ul>
 		</td>
+		<td>
+			{% assign ruleId = rule.id | remove: "/rules/" %}
+			{% assign implementations_count = site.data.implementations[ruleId] %}
+			<a href="{{ site.baseurl }}/pages/implementations.html">
+				{% if implementations_count == Nil %}
+					NEW
+				{% endif %}
+				{% if implementations_count == 0 %}
```

Comment:
Why not use `or`, DRY up the code?
`{% if implementations_count == Nil or implementations_count == 0  %}`